js/packages/phoenix-client/src/index.ts

```diff
@@ -1,3 +1,24 @@
-export * as phoenix from "./client/services.gen";
-export * as types from "./client/types.gen";
-export * as schemas from "./client/schemas.gen";
+import createOpenApiClient, { ClientOptions } from "openapi-fetch";
+import type {
+  paths as oapiPathsV1,
+  components as oapiComponentsV1,
+  operations as oapiOperationsV1,
+} from "./lib/api/v1.d.ts";
+
+type pathsV1 = oapiPathsV1;
+type componentsV1 = oapiComponentsV1;
+type operationsV1 = oapiOperationsV1;
+
+export type Types = {
+  V1: {
+    paths: pathsV1;
+    components: componentsV1;
+    operations: operationsV1;
+  };
+};
+
+export const createClient = (
+  options: ClientOptions = { baseUrl: "http://localhost:6006" },
```

Comment:
we should support the same environment variables here

Reply:
Good call, although I am a bit wary because it makes the code less isomorphic. I can do some feature detection on process.env probably so that this code still runs in all runtimes